src/assets/css/librarybrowser.css

```
@@ -1062,6 +1052,31 @@ div:not(.sectionTitleContainer-cards) > .sectionTitle-cards {
     }
 }
 
+@media only screen and (min-width: 1024px) {
+    .detailSticky {
+        background: transparent;
+        position: absolute;
+        width: 93.4%;
+        top: 10em;
+    }
+    img.itemDetailImage {
+        margin-top: -8em;
+    }
+    .detailPageContent {
+        width: 69.8%;
+        float: right;
+    }
+}
+
+.itemBackdrop:after {
+    content: "";
+    width: 100%;
+    height: 100%;
+    background: #000000ad;
+    background: rgba(0, 0, 0, 0.65)
```

1076:21  ✖  Missed semicolon   CssSyntaxError
```suggestion
    background: rgba(0, 0, 0, 0.65);
```